manifests/overlays/dev/kustomization.yaml

```diff
@@ -3,11 +3,49 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base
-patches:
+patchesJSON6902:
```

Comment:
Isnt patchesJSON6902 an older version of patches?
why is it picked here

Reply:
so they have some difference
https://github.com/kubernetes-sigs/kustomize/issues/2705#issuecomment-659012281